english/light-up/light-up.md

```diff
@@ -1,2 +1,17 @@
 Quick as lightning, you manage to duck the marshmallow hand.
-And with the lighter, you light up one end of the rope and push the other end into the wall.
\ No newline at end of file
+And with the lighter, you light up one end of the rope and 
+push the other end into the wall.
+
+The hand reaches for you again. You gab for your leather 
+coat and just in time you wrap it around the hand
+and stick the sleeves into the softened marshmallow.
+The hand is squirming to get free. One sleeve come free!
+
+A hole starts to form near the burning rope. This is your 
+chance to get free! The other sleeve has become free!
+The had is squirming to get the coat off.
+Hurry!
+
+Do you:
+
+[Grab the lighter and fork and jump throught the hole](grab_tools_and jump/grab_and_jump.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.